packages/amazonq/.changes/next-release/Feature-474727b5-ec93-4faa-ad2c-ddc8b6b5a6af.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Filter out non-code files when preparing for feature dev"
```

Comment:
please review https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog . then re-word this.